src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,719 @@
+    expense_vehicle_type:
+      allOf:
+        - $ref: ./fields.yaml#/vehicle_type
+      description: |
+        Vehicle type targeted by this policy.
+    expense_annual_distance_max:
+      type: number
+      example: 1200
+      description: |
+        Maximum travel distance allowed, above which policy will be violated.
+    employee_level_ids:
+      type: array
+      description: |
+        Employee level ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - lvl1sdfad
+        - lvl2sdfas
+    employee_level_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_custom_fields:
+      allOf:
+        - $ref: './fields.yaml#/custom_fields'
+      description: |
+        Custom fields
+    employee_custom_fields_op:
+      type: object
+      description: |
+        Operator for the employee_custom_fields.
+      properties:
+        field_name:
+          type: string
+    employee_department_ids:
+      type: array
+      description: |
+        Employee department ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - dept1asdfa
+        - dept2adfad
+    employee_department_id_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_locations:
+      description: |
+        Location of employees targeted by this policy.
+      type: array
+      items:
+        $ref: './fields.yaml#/location'
+    employee_locations_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_user_ids:
+      type: array
+      description: |
+        Employee user ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - usabcds1
+        - usabcds2
+    employee_user_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_titles:
+      type: array
+      description: |
+        Employee titles targeted by this policy.
+      items:
+        type: string
+      example:
+        - CEO
+        - Manager
+    employee_titles_op:
+      $ref: ./fields.yaml#/in_operator
+    action_flag:
+      type: boolean
+      description: |
+        Whether to flag the expense when it violates policy.
+      example: True
+    action_show_warning:
+      type: boolean
+      description: |
+        Whether to show warning to employee when the expense that they are creating violates the policy.
+      example: True
+    action_make_unreportable:
+      type: boolean
+      description: |
+        Whether to make the expense unreportable when it violates policy. <br>
+        (Expense will be created but employee will not be able to add it to report).
+      example: True
+    action_remove_employee_approver1:
+      type: boolean
+      description: |
+        Whether to remove the primary approver from the report when the expense violates policy.
+      example: True
+    action_add_employee_approver2:
+      type: boolean
+      description: |
+        Whether to add secondary approver 1 as approver to report when the expense violates policy.
```

Comment:
If this is set to true, then the employee's second approver will be added to the report that contains expense(s) violating this policy rule.